web/src/features/posthog-analytics/usePostHogClientCapture.ts

```diff
@@ -59,7 +59,13 @@ const events = {
     "version_delete_submit",
   ],
   session_detail: ["publish_button_click"],
-  eval_config: ["delete", "new_form_submit", "new_form_open"],
```

Comment:
nit/question: unsure if we "lose" historic data if we rename an event, often in analytics the event names are treated as uuids, so we'd have a new reporting after the rename. Either way, no biggy in this case, would just be good to know/keep in mind

Reply:
We are breaking in such that there is no more `delete` events, but only `toggling`. In what way do you mean we are 'losing' events? 🤔

Reply:
I think there is no other way than to extend by new events by having toggles now instead of deletes 👍🏾